src/login/70-uaccess.rules.in

```diff
@@ -63,7 +63,8 @@ ENV{ID_PDA}=="?*", TAG+="uaccess"
 # Programmable remote control
 ENV{ID_REMOTE_CONTROL}=="1", TAG+="uaccess"
 
-# joysticks
+# Gaming devices
+SUBSYSTEM=="input", ENV{ID_GAME_CONTROLLER}=="?*", TAG+="uaccess"
 SUBSYSTEM=="input", ENV{ID_INPUT_JOYSTICK}=="?*", TAG+="uaccess"
```

Comment:
given your first commit implied ID_GAME_CONTROLLER if ID_INPUT_JOYSTICK is set, you can drop the second line, no?

Reply:
Yes, I think so.

Reply:
`SUBSYSTEM=="input|hidraw"` ?